### PR TITLE
Update mpadded.json: support of lspace, width, height

### PR DESCRIPTION
**Pull request by Yaffle**
lspace, voffset, width, height has the same support info, so not sure if it is possible to use single row here.

https://bugs.webkit.org/show_bug.cgi?id=155792
https://bugs.webkit.org/show_bug.cgi?id=85730
https://github.com/WebKit/webkit/blob/master/Source/WebCore/rendering/mathml/RenderMathMLPadded.cpp#L98

### CODE REVIEW

File: mathml/elements/mpadded.json [hunk 1]
```diff
@@ -221,8 +221,157 @@
                 "version_added": false
               },
               "safari": {
+                "version_added": 10,
```
Comment (ExE-Boss) [inline mathml/elements/mpadded.json]:
```suggestion
                "version_added": "10",
```

File: mathml/elements/mpadded.json [hunk 2]
```diff
@@ -221,8 +221,157 @@
...
+                "partial_implementation": true,
+                "notes": "Relative and pseudo-unit values are not supported, see <a href='https://bugs.webkit.org/show_bug.cgi?id=85730'>bug 85730</a>."
```
Comment (ExE-Boss) [inline mathml/elements/mpadded.json]:
```suggestion
                "notes": "Relative and pseudo-unit values are not supported, see <a href='https://webkit.org/b/85730'>bug 85730</a>."
```

File: mathml/elements/mpadded.json [hunk 3]
```diff
@@ -221,8 +221,157 @@
...
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": 10,
```
Comment (ExE-Boss) [inline mathml/elements/mpadded.json]:
```suggestion
                "version_added": "10",
```

File: mathml/elements/mpadded.json [hunk 4]
```diff
@@ -221,8 +221,157 @@
...
+                "partial_implementation": true,
+                "notes": "Relative and pseudo-unit values are not supported, see <a href='https://bugs.webkit.org/show_bug.cgi?id=85730'>bug 85730</a>."
```
Comment (ExE-Boss) [inline mathml/elements/mpadded.json]:
```suggestion
                "notes": "Relative and pseudo-unit values are not supported, see <a href='https://webkit.org/b/85730'>bug 85730</a>."
```

File: mathml/elements/mpadded.json [hunk 5]
```diff
@@ -221,8 +221,157 @@
...
+                "version_added": false
+              },
+              "opera": {
                 "version_added": false
               },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": 10,
```
Comment (ExE-Boss) [inline mathml/elements/mpadded.json]:
```suggestion
                "version_added": "10",
```

File: mathml/elements/mpadded.json [hunk 6]
```diff
@@ -221,8 +221,157 @@
...
+                "partial_implementation": true,
+                "notes": "Relative and pseudo-unit values are not supported, see <a href='https://bugs.webkit.org/show_bug.cgi?id=85730'>bug 85730</a>."
```
Comment (ExE-Boss) [inline mathml/elements/mpadded.json]:
```suggestion
                "notes": "Relative and pseudo-unit values are not supported, see <a href='https://webkit.org/b/85730'>bug 85730</a>."
```

File: mathml/elements/mpadded.json [hunk 7]
```diff
@@ -221,8 +221,157 @@
...
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": 10,
```
Comment (ExE-Boss) [inline mathml/elements/mpadded.json]:
```suggestion
                "version_added": "10",
```

File: mathml/elements/mpadded.json [hunk 8]
```diff
@@ -221,8 +221,157 @@
...
+                "partial_implementation": true,
+                "notes": "Relative and pseudo-unit values are not supported, see <a href='https://bugs.webkit.org/show_bug.cgi?id=85730'>bug 85730</a>."
```
Comment (ExE-Boss) [inline mathml/elements/mpadded.json]:
```suggestion
                "notes": "Relative and pseudo-unit values are not supported, see <a href='https://webkit.org/b/85730'>bug 85730</a>."
```

**Comment by Yaffle** [comment]
@ExE-Boss , thanks, but I didn't find the "batch commit" possibility.
I am trying to `git reset` and make a single commit...

**Comment by ExE-Boss** [comment]
There should be an “Add to batch” button which is disabled on this page, but enabled on [the files tab](https://github.com/mdn/browser-compat-data/pull/4070/files).

Note that it might be better to just ignore it now (unless you intend to do an interactive `git rebase` to squash all the commits into one).

**Comment by Elchi3** [comment]
Hey there, thanks for the updates. I don't think this is sorted alphabetically yet. Here's what I'm seeing:
- mpadded
  - __compat
  - href
  - mathbackground
  - mathcolor
  - height 
  - lspace
  - voffset
  - width

I would expect:
- mpadded
  - __compat
  - href
  - height 
  - lspace
  - mathbackground
  - mathcolor
  - voffset
  - width

**Comment by Yaffle** [comment]
Other attributes are not mpadded specific

File: mathml/elements/mpadded.json [hunk 9]
```diff
@@ -49,6 +49,55 @@
...
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "edge_mobile": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": true
```
Comment (vinyldarkscratch) [inline mathml/elements/mpadded.json]:
```suggestion
                "version_added": "1"
```

File: mathml/elements/mpadded.json [hunk 10]
```diff
@@ -49,6 +49,55 @@
...
+              },
+              "firefox_android": {
+                "version_added": true
```
Comment (vinyldarkscratch) [inline mathml/elements/mpadded.json]:
```suggestion
                "version_added": "4"
```

File: mathml/elements/mpadded.json [hunk 11]
```diff
@@ -96,6 +145,55 @@
...
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "edge_mobile": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": true
```
Comment (vinyldarkscratch) [inline mathml/elements/mpadded.json]:
```suggestion
                "version_added": "1"
```

File: mathml/elements/mpadded.json [hunk 12]
```diff
@@ -96,6 +145,55 @@
...
+              },
+              "firefox_android": {
+                "version_added": true
```
Comment (vinyldarkscratch) [inline mathml/elements/mpadded.json]:
```suggestion
                "version_added": "4"
```

File: mathml/elements/mpadded.json [hunk 13]
```diff
@@ -221,8 +319,59 @@
...
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "edge_mobile": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": true
```
Comment (vinyldarkscratch) [inline mathml/elements/mpadded.json]:
```suggestion
                "version_added": "1"
```

File: mathml/elements/mpadded.json [hunk 14]
```diff
@@ -221,8 +319,59 @@
...
+              },
+              "firefox_android": {
+                "version_added": true
```
Comment (vinyldarkscratch) [inline mathml/elements/mpadded.json]:
```suggestion
                "version_added": "4"
```

**Pull request merged**